src/plugins/discover/public/application/embeddable/search_template_datagrid.html

```diff
@@ -1,16 +1,16 @@
 <discover-grid
   class="dscDiscoverGrid"
   columns="columns"
-  data-description="{{description}}"
-  data-shared-item
-  data-title="{{sharedItemTitle}}"
+  search-description="description"
+  search-title="sharedItemTitle"
+  data-test-subj="embeddedSavedSearchDocTable"
```

Comment:
Why removing `data-shared-item` here? I thought this was needed for reporting. Works fine without it, I'm just not sure I understand.

Reply:
added a note where I placed it (migrated all `data-` in the react component)